package.json

```diff
@@ -14,7 +14,7 @@
     "lint": "eslint src/ example/",
     "test": "echo 'There are no tests'",
     "flow": "flow",
-    "transpile": "rm -rf lib && babel src --out-dir lib --copy-files",
+    "transpile": "rm -rf lib && babel src --out-dir lib --copy-files && cp libdef.flow.js lib/",
```

Comment:
hvis du legger fila i src, så vil den automatisk bli kopiert til lib. Da er det ikke nødvendig med egen cp

Reply:
Den er grei, kan legge i src

Reply:
Nei, kan ikkje legge i flow, for då kjører babel på `libdef.flow.js`, og stripper vekk alt :P

Reply:
kan den ligge i flow-types mappen?